package.json

```diff
@@ -45,5 +45,8 @@
     "packages/*",
     "integrations/*",
     "playwright"
-  ]
+  ],
+  "dependencies": {
+    "react-aria": "^3.35.1"
+  }
```

Comment:
this is wrong
can you say why?

Reply:
This dependency should be defined in "packages/editor/package.json"?

Reply:
exactly.